KNOWLEDGE.md

```
@@ -11,3 +11,41 @@
 **Tags**: <空白区切りキーワード>
 **Rule**: <教訓本文>
 -->
+
+### Named arguments for builtins: generic parse, builtin-only dispatch
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)